### DISCUSSION TITLE
ScrollIntoView function of ListView does not work for custom ItemsPanel set in application level

### DISCUSSION THREAD

**Discussion by SubashiniMahendran** [microsoft/microsoft-ui-xaml#7434] [Q&A]
Hi Team,

I have set StackPanel as ItemsPanel for ListView control in application level. Then when adding new items at run-time, I need to bring the newly added item into view. So, I have tried to use ScrollIntoView function of ListView. But, it does not work properly for custom ItemsPanel set at sample level. Please find the demo sample below and replication procedure below:

**Demo:**
[Scrolling-issue.zip](https://github.com/microsoft/microsoft-ui-xaml/files/7458376/Scrolling-issue.zip)

**Replication procedure:**
1. Run the sample.
2. Try to add new item at run-time. I have used ScrollIntoView method to bring newly added item into view.
3. Note, newly added item not come into view even after using ScrollIntoView function of ListView control.

Can you please conform whether it is issue or behavior. If it is behavior, can you please provide any solution how to bring an ListViewItem into view.

**Comment by StephenLPeters** [comment]
@RBrid thoughts?

**Comment by shelllet** [comment]
upgrade to preview-3,
mouse scroll button don't work when view has scrollbar
![image](https://user-images.githubusercontent.com/16729017/141255603-3f3197d7-b522-4c14-96a3-b96b468b37cc.png)

**Comment by SubashiniMahendran** [comment]
Hi Team, 

Any update on this issue?

**Comment by StephenLPeters** [comment]
@ranjeshj or @RBrid any immediate thoughts?
